packages/TorneloScoresheet/src/components/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,51 @@
+import { View } from 'react-native';
+import { SvgProps } from 'react-native-svg';
+import { colours } from '../../style/colour';
+import PrimaryText, { FontWeight } from '../PrimaryText/PrimaryText';
+import RoundedView from '../RoundedView/RoundedView';
+import React from 'react';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { styles } from './style';
+
+export enum ButtonHeight {
+  SINGLE,
+  DOUBLE,
+}
+export type ActionButtonProps = {
+  Icon: React.FC<SvgProps>;
+  onPress: () => void;
+  text: string;
+  buttonHeight: ButtonHeight;
+};
+
+const SINGLE_UNIT_HEIGHT = 70;
+
+const ActionButton: React.FC<ActionButtonProps> = ({
+  Icon,
+  text,
+  buttonHeight,
+  onPress,
+}) => {
+  return (
+    <TouchableOpacity onPress={() => onPress()} style={{}}>
+      <RoundedView
+        style={[
+          styles.buttonContainer,
+          {
+            height: (buttonHeight + 1) * SINGLE_UNIT_HEIGHT,
+            width: SINGLE_UNIT_HEIGHT,
+          },
+        ]}>
+        <View
+          style={{ height: '70%', display: 'flex', justifyContent: 'center' }}>
```

Comment:
just double checking if it's ok for the style to be in here instead of the style file?

Reply:
it is very sinful and an accident

Reply:
We'll discuss your punishment at the next standup ;)